docker/build_scripts/fixup-mirrors.sh

```diff
@@ -3,12 +3,3 @@
 
 # Stop at any error, show all commands
 set -exuo pipefail
```

Comment:
Sounds like the rest of that file could be deleted too.

Reply:
Thanks for the suggestion.

The whole fixup-mirrors could be removed however:
- This line is not manylinux2010 specific so not removed. It's part of the skeleton of every script in here.
- I'd rather keep the file as it will be needed at some point (I'd rather have it kept around rather than dig through history).